tests/test_schema.py

```diff
@@ -371,6 +371,17 @@ def test_schema_bad_prop_06(self):
             abstract link foo extending foo;
         """
 
+    @tb.must_fail(errors.InvalidReferenceError,
+                  "object type or alias 'std::str' does not exist"):
+    def test_schema_bad_prop_07(self):
+        """
+            type Person {
+                required property name := str {
```

Comment:
Maybe use a computed link to have a non-failing test?

Reply:
That's still an error, because `including a shape on schema-defined computed links is not yet supported`